boards/Protectli/VP66xx/v0.9.0-results.csv

```diff
@@ -0,0 +1,192 @@
+Boot FreeBSD-RELEASE from disk after cold-boot,BFB001.001,SKIP,PASS
+Boot FreeBSD-RELEASE from disk after warm-boot,BFB002.001,SKIP,PASS
+Boot FreeBSD-RELEASE from disk after reboot,BFB003.001,SKIP,PASS
+Boot OPNsense (serial output) from disk after cold-boot,BOS001.001,SKIP,PASS
+Boot OPNsense (serial output) from disk after warm-boot,BOS002.001,SKIP,PASS
+Boot OPNsense (serial output) from disk after reboot,BOS003.001,SKIP,PASS
+Boot OPNsense (VGA output) from disk after cold-boot,BOV001.001,SKIP,PASS
+Boot OPNsense (VGA output) from disk after warm-boot,BOV002.001,SKIP,PASS
+Boot OPNsense (VGA output) from disk after reboot,BOV003.001,SKIP,PASS
+Boot pfSense LTS CE (serial output) from disk after cold-boot,BPS001.001,SKIP,PASS
+Boot pfSense LTS CE (serial output) from disk after warm-boot,BPS002.001,SKIP,PASS
+Boot pfSense LTS CE (serial output) from disk after reboot,BPS003.001,SKIP,PASS
+Boot pfSense LTS CE (VGA output) from disk after cold-boot,BPV001.001,SKIP,PASS
+Boot pfSense LTS CE (VGA output) from disk after warm-boot,BPV002.001,SKIP,PASS
+Boot pfSense LTS CE (VGA output) from disk after reboot,BPV003.001,SKIP,PASS
+Boot Windows 11 from disk after cold-boot,BWI001.001,SKIP,PASS
+Boot Windows 11 from disk after warm-boot,BWI002.001,SKIP,PASS
+Boot Windows 11 from disk after reboot,BWI003.001,SKIP,PASS
+Serial boot measure: coreboot booting time after cold boot,CBMEM001.001,PASS,PASS
+Serial boot measure: coreboot booting time after warm boot,CBMEM002.001,PASS,PASS
+Serial boot measure: coreboot booting time after system reboot,CBMEM003.001,PASS,PASS
+CPU temperature without load (Ubuntu),CPT001.001,FAIL,PASS
+CPU temperature after stress test (Ubuntu),CPT002.001,FAIL,PASS
+CPU not stuck on initial frequency (Ubuntu),CPF001.001,PASS,PASS
+CPU not stuck on initial frequency (Windows),CPF001.002,PASS,PASS
+CPU runs on expected frequency (Ubuntu),CPF002.001,PASS,PASS
+CPU runs on expected frequency (Windows),CPF002.002,PASS,PASS
+CPU with load runs on expected frequency (Ubuntu),CPF004.001,PASS,PASS
+CPU with load runs on expected frequency (Windows),CPF004.002,PASS,PASS
+Change the UUID,DCU001.001 ,PASS,PASS
+Change the serial number,DCU002.001,PASS,PASS
+Change the bootsplash logo,DCU003.001 ,PASS,PASS
+Netboot is available,NBT001.001,PASS,PASS
+OS Selection & Utilities is available,NBT002.001,PASS,PASS
+iPXE boot is available,NBT003.001,PASS,PASS
+iPXE shell is available,NBT004.001,PASS,PASS
+iPXE shell works correctly,NBT005.001,PASS,PASS
+Advanced option is available,NBT006.001,PASS,PASS
+Change netboot URL works correctly,NBT007.001,PASS,PASS
+USB detection after coldboot,UDT001.001,PASS,PASS
+USB detection after warmboot,UDT001.002,PASS,PASS
+USB detection after system reboot,UDT001.003,PASS,PASS
+USB detect and boot after coldboot,UBT001.001,FAIL,FAIL
+USB detect and boot after warmboot,UBT002.001,FAIL,FAIL
+USB detect and boot after system reboot,UBT003.001,FAIL,FAIL
```

Comment:
Why it doesn't work? Detection works but not booting? Was there no OS on USB or what?

Reply:
Here again it is the situation of joining the results. In RC6 USB devices were not detected after coldboot and boot to os failed after coldboot, warmboot and reboot. In RC7 only usb detection after coldboot was tested and boot to os was not.

Reply:
I cannot see the test anymore in the scope, so the comment is probably not valid.